pkg/new-ui/v1beta1/frontend/package-lock.json

```diff
@@ -1,8 +1,15797 @@
 {
```

Comment:
Why the `package-lock` was re-generated ?

Reply:
<img src="https://user-images.githubusercontent.com/52827441/136673850-d86b038d-20ad-4151-a937-3a0beb5a6ae2.png" width="650"/>

I think it's just how a newer version of NPM (over 7) works. (ref: [stackoverflow](https://stackoverflow.com/a/68479189/11717184))

Reply:
Does it mean that we have to update `npm` version in our UI development guide: https://github.com/kubeflow/katib/tree/master/pkg/new-ui/v1beta1#requirements ?

Reply:
Can we know the progress of this PR? 😄 

I think I can just remove the change of the package-lock.json from this PR, since this PR is nothing to do with the dependency tree.

Reply:
@seong7 apologies for the huge delay on this one. I'll take a closer look either later today or tomorrow tops.

Reply:
I've left some comments on how to handle the `package-lock.json`. @seong7 indeed let's drop the changes in that file in this PR.

I'm looking at the other changes now, for which I'll leave discrete review comments in different messages later on if needed.